packages/interaction-tracking/src/InteractionTracking.js

```diff
@@ -0,0 +1,341 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import invariant from 'shared/invariant';
+import {
+  enableInteractionTracking,
+  enableInteractionTrackingObserver,
```

Comment:
I don’t get why these feature flags exist

Reply:
Or rather, I get why they exist for React but not why they are used by this package. Is it because we’re still inlining it instead of publishing as a separate module?

Reply:
The `enableInteractionTracking` is used to make the production bundle no-op for interaction tracking (since nothing consumes that data).

The `enableInteractionTrackingObserver` flag is maybe not necessary, but I was using it when the subscription stuff was more obnoxious so that we could turn it off entirely if we decided to (e.g. for open source). I could remove this flag if we want.